### ISSUE TITLE
Can't create custom token filter

### ISSUE THREAD

**Issue by chr7s** [phillbaker/terraform-provider-elasticsearch#192]
I'm trying to create an `elasticsearch_index` but I don't find a way to define my custom token filter.

Here is the `analysis` part of my mapping. The `tokenizer` and `analyzer` part can be implemented using the `analysis_tokenizer` and `analysis_analyzer`. But I can't find anything like an `analysis_filter`
```
"analysis": {
      "analyzer": {
        "my_autocomplete_analyzer": {
          "tokenizer": "my_autocomplete_tokenizer",
          "filter": [
            "lowercase"
          ]
        },
        "my_shingle_analyzer": {
          "tokenizer": "standard",
          "filter": [
            "lowercase",
            "my_filter_shingle"
          ]
        }
      },
      "filter": {
        "my_filter_shingle":{
          "type":"shingle",
          "max_shingle_size":2,
          "min_shingle_size":2,
          "output_unigrams": false
        }
      }, 
      "tokenizer": {
        "my_autocomplete_tokenizer": {
          "type": "edge_ngram",
          "min_gram": 1,
          "max_gram": 10,
          "token_chars": [
            "letter",
            "digit"
          ]
        }
      }
    }
```

Is there a way to do this with this provider?

**Comment by phillbaker** [comment]
Hello, thanks for opening an issue! 

This isn't implemented yet, but it'd be easy to submit a PR, It'd be very similar to https://github.com/phillbaker/terraform-provider-elasticsearch/commit/611975d85b4385e236b772ad717e77ee6929f0e2

**Comment by phillbaker** [comment]
Fixed in 51fd1ce2486055861cb6e59e3117f392be7850b6
